backend/dataall/modules/dataset_sharing/common/enums.py

```diff
@@ -1,3 +1,4 @@
+
 from dataall.base.api.constants import GraphQLEnumMapper
 
 
```

Comment:
Why did we move it out of `api`? All other GraphQLEnumMapper enums are inside the `api` subpackage of the module